jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/MessagesTable.kt

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.sqs.toolwindow
+
+import com.intellij.openapi.Disposable
+import com.intellij.ui.ScrollPaneFactory
+import com.intellij.ui.TableSpeedSearch
+import com.intellij.ui.table.TableView
+import com.intellij.util.ui.ListTableModel
+import software.amazon.awssdk.services.sqs.model.Message
+import software.aws.toolkits.resources.message
+import javax.swing.JComponent
+import javax.swing.JTable
+
+class MessagesTable : Disposable {
```

Comment:
This does not appear to be registered to a parent disposable, but also I do not see it acting as a parent either. Looks like you can remove the Disposable interface